pyproject.toml

```diff
@@ -42,10 +42,12 @@ dependencies = [
     "typing_extensions>=3.10.0.0",
     "packaging>=20.9",
     "pandas>=1.4.2",
+    "Pillow>=10.2.0",
     "numpy>=1.22.4",
     "Babel>=2.13.1",
     "mizani>=0.9.3",
-    "webcolors",
+    "selenium>=4.18.1",
```

Comment:
I wonder if it's worth making this (and Pillow) an optional dependency? We could make a set of optional dependencies called `web` or similar, and queue users to `pip install 'great_tables[html]'` or `pip install 'great_tables[web]'`

(I'm def open to us keeping it as required!)

Reply:
I moved those deps into `extra`. Will document in README about the different pip install options.